.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+    release:
+        types:
+            - created
+jobs:
+    test:
+        name: Test
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Set up Python 3.9
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  python -m pip install .
+                  python -m pip install -r requirements-dev.txt
+            - name: Test
+              run: |
+                  python -m pytest tests/test_get_dataloaders.py
+                  python -m pytest tests/test_metadata_utils.py
```

Comment:
`test_train.py` took about 5 minutes to complete so excluded for now.

Reply:
Thank you very much for this addition!

Reply:
@SaulLu You're very welcome! I hope this will help a little without too much intrusion...